### PR TITLE
Minor fixes to "conditionals" section of bash docs

### PR DESCRIPTION
**Pull request by ilan1**

### CODE REVIEW

File: bash.html.markdown [hunk 1]
```diff
@@ -97,12 +98,12 @@ echo "Always executed" && echo "Only executed if first command does NOT fail"
 # To use && and || with if statements, you need multiple pairs of square brackets:
 if [ $Name == "Steve" ] && [ $Age -eq 15 ]
 then
-    echo "This will run if $Name is Steve AND $Age is 15."
+    echo 'This will run if $Name is Steve AND $Age is 15.'
```
Comment (vendethiel) [inline bash.html.markdown]: I think those two were meant to be double quotes, actually

Comment (ilan1) [inline bash.html.markdown]: OK, my mistake then

Comment (vendethiel) [inline bash.html.markdown]: @ilan1 can you please fix it so that I can merge? :)

**Comment by vendethiel** [comment]
@ilan1 any update?:)

**Comment by vendethiel** [comment]
Going to close this one for now :). Feel free to ping us here if you find some time working on it again! Thanks.

**Pull request closed without merging**